modules/json/tests/test_dot_notation.c

```diff
@@ -93,53 +72,59 @@ assert_dot_notation_eval_fails(const gchar *input_json, const gchar *subscript)
 
   input = compile_json(input_json);
   sub = json_extract(input, subscript);
-  assert_null(sub, "extracted JSON is not NULL as expected, json=%s, subscript=%s", input_json, subscript);
+  cr_assert_null(sub, "extracted JSON is not NULL as expected, json=%s, subscript=%s", input_json, subscript);
   json_object_put(input);
 }
 
-static void
-test_dot_notation_eval_empty_subscript_returns_the_object(void)
+void setup(void)
+{
+  app_startup();
+}
+
+void teardown(void)
+{
+  app_shutdown();
+}
+
+Test(json, test_dot_notation_eval_empty_subscript_returns_the_object, .init = setup, .fini = teardown)
 {
   assert_dot_notation_eval_equals("{'foo': 'bar'}", "", "{'foo': 'bar'}");
 }
 
-static void
-test_dot_notation_eval_fails_with_an_identifier_that_doesnt_start_with_a_letter(void)
+Test(json, test_dot_notation_eval_fails_with_an_identifier_that_doesnt_start_with_a_letter, .init = setup,
```

Comment:
You could define a `TestSuite`, in which case you do not have to repeat the `.init = setup, .fini = teardown` part.

Reply:
fixed